modules/tinymce/src/core/main/ts/DragDropOverrides.ts

```diff
@@ -261,9 +263,61 @@ const blockIeDrop = function (editor: Editor) {
   });
 };
 
+// Block files being dropped within the editor to prevent accidentally navigating away
+// while editing. Note that we can't use the `editor.on` API here, as we want these
+// to run after the editor event handlers have run. We also bind to the document
+// so that it'll try to ensure it's the last thing that runs, as it bubbles up the dom.
+const blockUnsupportedFileDrop = (editor: Editor) => {
+  const preventFileDrop = (e: DragEvent) => {
+    if (!e.defaultPrevented) {
+      // Prevent file drop events within the editor, as they'll cause the browser to navigate away
+      const dataTransfer = e.dataTransfer;
+      if (dataTransfer && (Arr.contains(dataTransfer.types, 'Files') || dataTransfer.files.length > 0)) {
+        // TODO: Add an error notification in 5.5
```

Comment:
just checking: do we have a jira for this?

Reply:
No, see the comment in the description:
> Note: I'll create a follow-up PR once this is merged to port it to develop and add the notification for TinyMCE 5.5